tests/data/setup.py

```diff
@@ -14,6 +14,9 @@
 requirements = [
     'click>=5.0.0',
     'mock>=1.3.0',
+    'invenio-db[versioning]>=1',
+    'invenio-db[versioning,mysql]>=1',
```

Comment:
@david-caro When you re-push, please also add yourself to the `AUTHORS.rst` file, thanks!